docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
+        "June", "July", "August",
+        "September", "October", "November",
+        "December"
+    )
+    
+    // Sales data for different products and regions:
+    val salesLaptop = listOf(120, 130, 150, 180, 200, 220, 240, 230, 210, 190, 160, 140)
+    val salesSmartphone = listOf(90, 100, 110, 130, 150, 170, 190, 180, 160, 140, 120, 100)
+    val salesTablet = listOf(60, 70, 80, 90, 100, 110, 120, 110, 100, 90, 80, 70)
+    
+    // A data frame with columns for Month, Sales, and Product
+    val dfSales = dataFrameOf(
+        "Month" to months + months + months,
+        "Sales" to salesLaptop + salesSmartphone + salesTablet,
+        "Product" to List(12) { "Laptop" } + List(12) { "Smartphone" } + List(12) { "Tablet" },
+    )
+    ```
+
+    The data frame uses the `dataFrameOf()` function and includes the number of products (laptops, smartphones,
+    and tablets) sold in a 12-month period.
+
+3. Explore the data in your frame, for example, by finding the product and month with the highest sales:
+
+   ```none
+   dfSales.maxBy("Sales")
+   ```
+
+![Using DataFrame to visualize data](dataframe-output.png){width=500}
+
+## Charts
+
+You can create various charts directly in your Kotlin Notebook to visualize your data:
+
+1. Add the [Kandy](https://kotlin.github.io/kandy/welcome.html) plotting library to your notebook:
 
-## LaTeX markup
+   ```none
+   %use kandy
+   ```
 
-Mathematical formulas and equations can be beautifully rendered using LaTeX, a typesetting system widely used in academia.
+2. Use the same data frame and run the `plot()` function in the new cell:
 
-## Error and traceback
+   ```kotlin
+   dfSales.groupBy { Product }.plot {
+       bars {
+           // Access the data frame's columns used for the X and Y axes
+           x(Month)
+           y(Sales)
+           // Access the data frame's column used for categories and sets colors for these categories
+           fillColor(Product) {
+               scale = categorical(
+                   "Laptop" to Color.PURPLE,
+                   "Smartphone" to Color.ORANGE,
+                   "Tablet" to Color.GREEN
+               )
+               legend.name = "Product types"
+           }
+       }
+       // Customize the chart's appearance
+       layout.size = 1000 to 450
+       layout.title = "Yearly Gadget Sales Results"
+   }
+   ```
 
-When code contains errors, notebooks display an error message and a traceback, providing insights for debugging.
+![Using Kandy to render visualize data](kandy-output.png){width=700}
 
 ## What's next
 
-* Visualize data using the Kandy library
-* Work with csv and json data
-* Check out the list of supported libraries
\ No newline at end of file
+* [Visualize data using the DataFrame and Kandy libraries](data-analysis-visualization.md)
+* [Retrieve data from the CSV and JSON files](data-analysis-work-with-data-sources.md)
+* [Check out the list of supported libraries](data-science-libraries.md)
```

Comment:
Maybe we should get rid of "supported" word here, it creates the illusion you can only use them. In fact, you can use any JVM library